### ISSUE TITLE
New version 3.8 crashes edit.php

### ISSUE THREAD

**Issue by tamorley** [WordPress/gutenberg#9876]
Installed the new version 3.8 and cannot access edit.php to view all my posts. I get an http error 500.

Rollback to version 3.7 fixes the problem.

Using MH Magazine theme.

**Comment by designsimply** [comment]
The 500 error is a bit vague and indicates that the server encountered an unexpected condition that prevented it from fulfilling the request.

I tested Gutenberg 3.8 on a few different hosts and I have been unable to trigger a 500 error for the Posts list that appears when 3.8 is installed but goes away for 3.7. This probably means there is something specific to your setup which is causing a conflict but I am not sure what without more information.

Is there a chance you can find out more detail about the error message? Is there something more specific in the browser console at the time you open edit.php? Are you able to check your server logs to see if there are any related errors showing there?

I also searched online and this is one of the results that came up near the top and it has some notes about .htaccess files and making sure the PHP memory limit is set properly which may be helpful to try: https://www. .com/blog/tips-tricks/how-to-fix-the-500-internal-server-error-on-your-wordpress-website

**Comment by Taubin** [comment]
Site is running on Ubuntu 16.04 with NGINX:

```
$ nginx -V
nginx version: nginx/1.10.3 (Ubuntu)
built with OpenSSL 1.0.2g  1 Mar 2016
TLS SNI support enabled
configure arguments: --with-cc-opt='-g -O2 -fPIE -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2' --with-ld-opt='-Wl,-Bsymbolic-functions -fPIE -pie -Wl,-z,relro -Wl,-z,now' --prefix=/usr/share/nginx --conf-path=/etc/nginx/nginx.conf --http-log-path=/var/log/nginx/access.log --error-log-path=/var/log/nginx/error.log --lock-path=/var/lock/nginx.lock --pid-path=/run/nginx.pid --http-client-body-temp-path=/var/lib/nginx/body --http-fastcgi-temp-path=/var/lib/nginx/fastcgi --http-proxy-temp-path=/var/lib/nginx/proxy --http-scgi-temp-path=/var/lib/nginx/scgi --http-uwsgi-temp-path=/var/lib/nginx/uwsgi --with-debug --with-pcre-jit --with-ipv6 --with-http_ssl_module --with-http_stub_status_module --with-http_realip_module --with-http_auth_request_module --with-http_addition_module --with-http_dav_module --with-http_geoip_module --with-http_gunzip_module --with-http_gzip_static_module --with-http_image_filter_module --with-http_v2_module --with-http_sub_module --with-http_xslt_module --with-stream --with-stream_ssl_module --with-mail --with-mail_ssl_module --with-threads
```
```

$ php -v
PHP 7.2.9-1+ubuntu16.04.1+deb.sury.org+1 (cli) (built: Aug 19 2018 07:16:12) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.9-1+ubuntu16.04.1+deb.sury.org+1, Copyright (c) 1999-2018, by Zend Technologies
```

**Comment by designsimply** [comment]
Thank you for the detailed report @Taubin! 

Do you have access to your server's error logs, and can you see if there's any more information showing in them?

If you have a firewall setup on your server, can you check your firewall logs to see if there is a log for a related rule being triggered?

**Comment by tamorley** [comment]
This is what my error log says:

[13-Sep-2018 18:25:59 UTC] PHP Warning:  array_values() expects parameter 1 to be array, null given in /home/tmorley1988/public_html/wp-content/plugins/gutenberg/lib/client-assets.php on line 324
[13-Sep-2018 18:25:59 UTC] PHP Warning:  array_values() expects parameter 1 to be array, null given in /home/tmorley1988/public_html/wp-content/plugins/gutenberg/lib/client-assets.php on line 325
[13-Sep-2018 18:25:59 UTC] PHP Warning:  array_values() expects parameter 1 to be array, null given in /home/tmorley1988/public_html/wp-content/plugins/gutenberg/lib/client-assets.php on line 326
[13-Sep-2018 18:25:59 UTC] PHP Warning:  array_values() expects parameter 1 to be array, null given in /home/tmorley1988/public_html/wp-content/plugins/gutenberg/lib/client-assets.php on line 327

**Comment by youknowriad** [comment]
Do you still have the issues with WordPress 5.0 ? Were you able to figure this out?

**Comment by tamorley** [comment]
The issue resolved itself in the 4.9.? release and still works fine in 5.0.  Thanks

**Comment by youknowriad** [comment]
Awesome 👍
